web/tests/compose.test.js

```diff
@@ -323,10 +323,12 @@ test_ui("send_message", ({override, override_rewire, mock_template}) => {
 
     (function test_error_codepath_local_id_undefined() {
         let banner_rendered = false;
+        $.create("#stub-banner")
```

Comment:
I'm not totally sure if this is the right way to fix the zquery situation here, let me know :)